### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by ntqdinh-axonivy**

### CODE REVIEW

**Comment by ivy-sgi** [comment]
Please add the following sentence somewhere:  Include a task or a task switch element aftern an alternative.

**Comment by ivy-sgi** [comment]
Hi @ntqdinh-axonivy  Please use the standard logo: 
image

**Comment by ntqdinh-axonivy** [comment]
Hi @ivy-sgi 
I have updated the readme with more hints about setup steps and images of the process analyzer's diagram in the latest version.
Could you have a quick look and help us with the German translation for readme file?
Thanks

(I will update the logo in Axon Ivy market PR later.)

File: process-analyser-product/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- **Flexible Process Selection**
+: Well integrated with any process from the same security context to leverage the tool's robust features.
+- **Export Capabilities**
+: Export diagram and detailed Excel reports with frequency/duration statistics.
+
+## Demo
+
+- Select the PMV that matches the desired process and KPI type, then click the **"Show Statistic"** button. This will display a raw data visualized process diagram alongside an analyzed statistics table. *(In this version, the data is limited to **"DONE"** cases with fewer than **TWO** alternative elements in their process.)*
+
+![show statistic demo](show-statistic.gif)
```
Comment (phhung-axonivy) [inline process-analyser-product/README.md:22]: Could you please create a new folder 'images' and move all images to that folder?
Afterthat update the README (DE) files.

**Review by phhung-axonivy — CHANGES_REQUESTED** [review]
LGTM

**Review by phhung-axonivy — APPROVED** [review]
LGTM

**Pull request merged**